install/tools/setupMeshEx.sh

```diff
@@ -122,17 +110,16 @@ function istio_provision_certs() {
   if [[ -n "$NS" ]] ; then
     NS="-n $NS"
   fi
+
+  # shellcheck disable=SC2086
+  go run ./security/tools/generate_cert -client \
+    -host spiffee://cluster.local/${NS}/${SA} -mode citadel --out-cert cert-chain.pem --out-priv key.pem
+  echo "Generated cert-chain.pem and key.pem. It should be installed on /etc/certs"
+
   local B64_DECODE=${BASE64_DECODE:-base64 --decode}
   # shellcheck disable=SC2086
-  kubectl get $NS secret "$CERT_NAME" -o jsonpath='{.data.root-cert\.pem}' | $B64_DECODE   > root-cert.pem
+  kubectl get cm istio-security -n istio-system -o "jsonpath={.data.caTLSRootCert}" | $B64_DECODE   > root-cert.pem
   echo "Generated root-cert.pem. It should be installed on /etc/certs"
-  if [ "$ALL" == "all" ] ; then
```

Comment:
Removing $ALL only reference makes the linter choke on LINE 107, you should remove unused ALL variable